### ISSUE TITLE
[1.1.3 patch]: Timeouts in test runs

### ISSUE THREAD

**Issue by pranavkm** [aspnet/EntityFramework#8139]
1. DatabindingSqlServerTest.Adding_entity_to_state_manager_of_different_type_than_local_view_typ 	
2. DatabindingSqlServerTest.Entities_removed_from_the_local_view_are_marked_deleted_in_the_state 	
3. DatabindingSqlServerTest.DbSet_Local_does_not_call_DetectChanges 
4. TransactionSqlServerTest.SaveChanges_uses_explicit_transaction_and_does_not_rollback_on_failu
5. TransactionSqlServerTest.RelationalTransaction_can_be_commited(autoTransaction: True) 

Stack trace looks like this:

```
System.Data.SqlClient.SqlException : Timeout expired.  The timeout period elapsed prior to completion of the operation or the server is not responding.
ALTER DATABASE statement failed.
---- System.ComponentModel.Win32Exception : The wait operation timed out
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlCommand.RunExecuteNonQueryTds(String methodName, Boolean async, Int32 timeout, Boolean asyncWrite)
   at System.Data.SqlClient.SqlCommand.InternalExecuteNonQuery(TaskCompletionSource`1 completion, Boolean sendToPipe, Int32 timeout, Boolean asyncWrite, String methodName)
   at System.Data.SqlClient.SqlCommand.ExecuteNonQuery()
   at Microsoft.EntityFrameworkCore.SqlServer.FunctionalTests.TransactionSqlServerFixture.<CreateTestStore>b__2_0
```

**Comment by pranavkm** [comment]
cc @ajcvickers  \ @Eilon

**Comment by smitpatel** [comment]
While we are there, why the patch versioned 1.1.3 for EF?

**Comment by pranavkm** [comment]
What do you mean?

**Comment by Eilon** [comment]
For EF the actual version is 1.1.2.

**Comment by ajcvickers** [comment]
@pranavkm With regard to the branch, please use the branch we already created for 1.1.2. It has existed for some time and we have been checking in all our 1.1.2 fixes to that branch.

@Eilon These tests were fixed on 2.0, but this fix required a product change to allow the timeout to be set for the command used to create the database. This has low value for customers since it is unlikely that they will be doing rapid creation of multiple databases like we do in our tests. It is therefore not something we would normally take in a patch release. So we could just disable these tests (although I don't know that this is the complete list) or we could take the fix in the patch release even though it doesn't meet the bar by normal standards. /cc @AndriySvyryd

**Comment by pranavkm** [comment]
Sure, I can fix up the Coherence-Patch branches to point to the repo. Can you get someone to cherry-pick the NuGet.config changes from the `patch/*` branches on to the release branches?

Also, another test failure on Windows:
```
SqlServerMigrationsTest.Empty_Migration_Creates_Database 
System.InvalidOperationException : An exception has been raised that is likely due to a transient failure. If you are connecting to a SQL Azure database consider using SqlAzureExecutionStrategy.
---- System.Data.SqlClient.SqlException : A connection was successfully established with the server, but then an error occurred during the login process. (provider: Named Pipes Provider, error: 0 - No process is on the other end of the pipe.)
-------- System.ComponentModel.Win32Exception : No process is on the other end of the pipe
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.<ExecuteAsync>d__6`2.MoveNext() in C:\BuildAgent\work\d6337ac2abf63a4b\.repositories\EntityFramework\src\Microsoft.EntityFrameworkCore.SqlServer\Storage\Internal\SqlServerExecutionStrategy.cs:line 56
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
```

**Comment by ajcvickers** [comment]
@pranavkm I don't know what this means: "Can you get someone to cherry-pick the NuGet.config changes from the patch/* branches on to the release branches?" Can you be more specific?

**Comment by smitpatel** [comment]
I know what it means. I will do it.

**Comment by pranavkm** [comment]
There's a commit in each of `patch/1.0.5` and `patch/1.13` that updates the NuGet.config to use the new patch feeds:

* https://github.com/aspnet/EntityFramework/commit/ec3d967bc53dd18e5590686fe973b9910d0c486a
* https://github.com/aspnet/EntityFramework/commit/e976132e8fe03d54a5761b0aefcfbdd4cad8a3bc

Once we have these cherry-picked, we should be good to delete the two patch feeds in this repo.

**Comment by ajcvickers** [comment]
@smitpatel @pranavkm  Thanks!

**Comment by Eilon** [comment]
@ajcvickers - your call on the best way to mitigate these failures. If changing timeouts is impractical (or impossible without new features), then disabling is fine. We generally don't add new features in patches, and almost certainly not for just a test.

**Comment by ajcvickers** [comment]
@Eilon  I'm fine with disabling. @smitpatel Can you do this as well, since you're already working in these branches. 😸

**Comment by smitpatel** [comment]
I updated the rel branches. @pranavkm - If you updated the patch build to use branches rel/1.0.4 & rel/1.1.2, we can know delete the patch branches and also get better feedback about what is failing which needs to be disabled.

**Comment by pranavkm** [comment]
@smitpatel just saw `DatabindingSqlServerTest.DbSet_Local_does_not_call_DetectChanges` fail. We're building `rel/1.1.2` of EF. Could you verify if your changes were commited to this branch?

**Comment by smitpatel** [comment]
Latest run in CI for EF 1.1.2 passed. Fixed via 23e45fa9a326448d6a56e1ca2245d9e0588f1417 & 3681db266ac7985d560faa497f1c37ac859a5da6
